Packs/Code42/IncidentFields/incidentfield-Exfiltrated_Files.json

```diff
@@ -0,0 +1,38 @@
+{
+	"associatedToAll": true,
```

Comment:
field should be associated to the new incident type and not to all types.
also field should contain `"fromVersion": "5.0.0"`

same for all fields